logins.js

```diff
@@ -25,10 +25,16 @@ module.exports = {
                 return callback(err, null);
             }
             
+            if (body.trim().indexOf('<') === 0) {
```

Comment:
Nope. Since this is checking if the ```<``` is only in the first position, it can successfully detect html vs. json.

Html would look like: "```<..../>```"
JSON would look like: "```{....}```"